src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/View/AbstractViewHandler.cs

```diff
@@ -43,16 +43,28 @@ protected AbstractViewHandler(PropertyMapper mapper)
 
 		public object? NativeView => TypedNativeView;
 
-		public virtual void SetView(IView view)
+
+		public void SetView(IView view)
```

Comment:
Removed the virtual on `SetView`  as I wasn't sure if we wanted to allow the user to not call this code at all. I added other life cycle points below that users can use to setup/teardown.

I also liked using specific methods like `TearDownOldVirtualView` because we can make it so they are only called after a "not null" check which keeps out a bunch of `if null` checks inside the handlers